frontend/src/main/components/Commons/CommonsForm.js

```diff
@@ -158,9 +190,19 @@ function CommonsForm({ initialCommons, submitAction, buttonLabel = "Create" }) {
         </Form.Control.Feedback>
       </Form.Group>
 
+      <h4>
+        Health update formula
+      </h4>
+      {healthUpdateStrategiesDropdown("aboveCapacityHealthUpdateStrategy", "When above capacity",
+        initialCommons?.aboveCapacityHealthUpdateStrategy || healthUpdateStrategies?.defaultAboveCapacity
+      )}
```

Comment:
As a react component, this would be:

```jsx
<HealthUpdateStrategiesDropdown
  formName={}
  displayName={}
  initialValue={ healthUpdateStrategiesinitialCommons?.aboveCapacityHealthUpdateStrategy }
  healthUpdateStratgies=healthUpdateStrategies
/>
```

And you could move the implicit check for null with the backup value `healthUpdateStrategies?.defaultAboveCapacity` inside the component. (That way it only has to be tested once).